lib/mailchimp_api/resources/base.rb

```diff
@@ -33,6 +33,12 @@ def headers
 
         _headers || {}
       end
+
+      def exists?(id, options = {})
+        true if find id, options
```

Comment:
I forget, can we specify fields to return for *all* the endpoints? If so, we should probably just request the primary key back. Save money on bandwidth and what not.

Reply:
This rings a bell. Will look into it sometime today and update if it's possible.